Cargo.toml

```diff
@@ -1,11 +1,18 @@
 [package]
 name = "netctl-tray"
-version = "0.1.0"
-authors = ["Ponas <mykolas.peteraitis@gmail.com>"]
+version = "0.2.0"
+authors = ["Ponas <mykolas.peteraitis@gmail.com>", "Max Zinkus <zinkus@cs.jhu.edu>"]
 edition = "2018"
 
 [dependencies]
 qt_widgets = "0.3.0"
 qt_gui = "0.3.0"
 regex = "1.3.1"
-shells = "0.2.0"
\ No newline at end of file
+shells = "0.2.0"
+
+[[bin]]
+name = "netctl-tray"
+path = "src/main.rs"
```

Comment:
what's the purpose of this?

Reply:
This might be redundant now, I can remove it.